job_spec/OPERA_RTC_S1.yml

```diff
@@ -0,0 +1,45 @@
+OPERA_RTC_S1:
+  required_parameters:
+    - granules
+  parameters:
+    granules:
+      api_schema:
+        type: array
+        minItems: 1
+        maxItems: 1
+        example:
+          - S1_073251_IW2_20200128T020712_VV_2944-BURST
+        items:
+          anyOf:
+            - description: Sentinel-1 co-pol IW burst granule to process.
+              type: string
+              pattern: '^S1_\d{6}_IW\d_\d{8}T\d{6}_(VV|HH)_[\dA-F]{4}-BURST$'
+              minLength: 43
+              maxLength: 43
+              example: S1_073251_IW2_20200128T020712_VV_2944-BURST
+  cost_profiles:
+    EDC:
+      cost: 1.0
+    DEFAULT:
+      cost: 1.0
+  validators:
+    - check_opera_rtc_s1_date
+    - check_dem_coverage
+    # Static coverage check is the slowest because it involves a CMR query, so should come last
+    - check_opera_rtc_s1_static_coverage
+  steps:
+    - name: ''
```

Comment:
What's the reason for the name with empty string? I see other job types have this, while others have names.

Reply:
From what I remember, we decided on this convention when implementing the ability for a job spec to have multiple steps. The name of the job type gets prefixed onto the name of the step, e.g. `OPERA_RTC_S1_STEP1`. If you just want the name of the step to be the same as the name of the job, then you leave name empty.